demo/src/app/pages/demo/components/advanced-example-types.component.ts

```diff
@@ -14,77 +15,88 @@ export class AdvancedExamplesTypesComponent {
       name: "Leanne Graham",
       username: "Bret",
       email: "Sincere@april.biz",
-      comments: "Lorem ipsum dolor sit amet, ex dolorem officiis convenire usu."
+      comments: "Lorem ipsum dolor sit amet, ex dolorem officiis convenire usu.",
+      custom: '10|20'
```

Comment:
Ok let me look into this and I will try to work on this feature before updating the PR